pkg/providers/instancetype/instancetypes.go

```diff
@@ -184,7 +226,13 @@ func (p *Provider) getInstanceTypes(ctx context.Context) (map[string]*skewer.SKU
 		}
 	}
 
-	logging.FromContext(ctx).Debugf("%d SKUs remaining after filtering", len(instanceTypes))
+	if p.cm.HasChanged("instance-types", instanceTypes) {
+		// Only update instanceTypesSeqNun with the instance types have been changed
+		// This is to not create new keys with duplicate instance types option
+		atomic.AddUint64(&p.instanceTypesSeqNum, 1)
+		logging.FromContext(ctx).With(
+			"count", len(instanceTypes)).Debugf("discovered instance types")
```

Comment:
can we use a name other than count? writing queries against the logs would be useful to have a term other than count in the json we can query against. 


InstanceCount maybe?

Reply:
Wouldn't it be possible to discern context from other fields? I dislike the idea of having to come up with different names in different contexts ...

Reply:
yeah you could do something like 

FakeEventsTable
| where msg startswith "discovered instance types" 
| project count

but the problem is that count is also a reserved keyword in KQL and other languages. So you would have do to 


FakeEventsTable
| where msg startswith "discovered instance types" 
| project ['count']

for it to actually run iirc